CHANGES.md

```diff
@@ -1,6 +1,11 @@
 # tulip changelog
 
 
+## 1.3.1
```

Comment:
I noticed that `jtlv` was marked with milestone 1.4.0, but thought that we may want to include it in 1.3.1 since that version was to be the next one. I agree about bumping to 1.4.0 to signify the API incompatibility.